Data/DayOffRequestRepository.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Data;
+using Microsoft.EntityFrameworkCore;
+using stunning_robot_HR.Models;
+
+namespace stunning_robot_HR.Data
+{
+    public class DayOffRequestRepository : IDayOffRequestRepository, IDisposable
+    {
+        private stunning_robot_HRContext _context;
+
+        public DayOffRequestRepository(stunning_robot_HRContext context)
+        {
+            _context = context;
+        }
+
+        public IEnumerable<DayOffRequest> GetDayOffRequest()
+        {
+            return _context.DayOffRequests.ToList();
+        }
+
+        public DayOffRequest GetDayOffRequestById(int id)
+        {
+            return _context.DayOffRequests.Find(id);
```

Comment:
Another area where a very simple query might be hidden behind the high walls of a repo. Encapsulating it.